app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Media Ranker 2.0</title>
+
+  <%= stylesheet_link_tag    "application", "https://fonts.googleapis.com/css?family=Gudea:400,700|Hammersmith+One", media: "all", 'data-turbolinks-track' => true %>
+  <%= favicon_link_tag(source='owl.jpg', options={}) %>
+  <%= javascript_include_tag "application", 'data-turbolinks-track' => true %>
+  <%= csrf_meta_tags %>
+</head>
+
+<body>
+
+
+  <header class="app-header__header">
+    <h1>
+      <a href="/">Media Ranker</a>
+      <small>Ranking the Best of Everything</small>
+    </h1>
+
+    <nav class="app-header__nav">
+      <ul class="nav app-header__site-nav-container">
+        <li class="nav-item app-header__nav_item">
+          <a class="nav-link" href="/">View top media</a>
+        </li>
+        <li class="nav-item app-header__nav_item">
+          <a class="nav-link" href="/works">View all media</a>
+        </li>
+        <li class="nav-item app-header__nav_item">
+          <a class="nav-link" href="/works/new">Add a new work</a>
+        </li>
+        <li class="nav-item app-header__nav_item">
+          <a class="nav-link" href="/users">View all users</a>
+        </li>
+      </ul>
+
+      <ul class="nav app-header__user-nav-container">
+
+        <% if session[:user_id] %>
+          <li class="nav-item app-header__nav_item">
+            <a class="btn btn-primary"
```

Comment:
You don't need the anchor tag here.  Since `link_to` generates an anchor tag, you can style it with Bootstrap like this:

```erb
<%= link_to "Logged in as #{@user.username}", user_path(@user.id), class: "btn btn-primary" %>
```